packages/functional-tests/tests/signin/connectAnotherDevice.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect, newPagesForSync } from '../../lib/fixtures/standard';
+
+test.describe('connect_another_device', () => {
+  test('signup Fx Desktop, load /connect_another_device page', async ({
+    credentials,
+    target,
+    pages: { connectAnotherDevice },
+  }) => {
+    const { page, login } = await newPagesForSync(target);
```

Comment:
This could potentially be flaky because it is using the original browser. You would need to use 

```
const { page, login, connectAnotherDevice } = await newPagesForSync(target)
```